Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MarkdownTextBlock/InitialContent.md

```diff
@@ -273,6 +273,40 @@ results in:
 
 There are other methods of generating links that aren't appropriate for discussion-board style comments.  See the [Markdown Syntax](http://daringfireball.net/projects/markdown/syntax#link) if you're interested in more info.
 
+&nbsp;
+
+Relative links are also supported
+
+>\[Relative Link\]\(/Assets/Photos/Photos.json\)
+
+results in:
+
+>[Relative Link](/Assets/Photos/Photos.json)
+
+&nbsp;
+
+>\[Relative Link 2\]\(../Photos/Photos.json\)
+
+results in:
+
+>[Relative Link 2](../Photos/Photos.json)
+
+**Note:** Relative Links has to be Manually Handled in `LinkClicked` Event.
```

Comment:
There's a new note syntax for markdown docs.

`> [!NOTE]`

Reply:
@michael-hawker I thought that was for control documentation. This is the initial content that loads when MarkdownTextblock loads

Reply:
Yes, sorry, missed this was in the sample app.